spring-cloud-dataflow-configuration-metadata/src/main/java/org/springframework/cloud/dataflow/configuration/metadata/container/ContainerImage.java

```diff
@@ -55,10 +55,13 @@
 	// and dashes. A tag name may not start with a period or a dash and may contain a maximum of 128 characters.
 	// (https://dockr.ly/3chhQZF)
 	private static final Pattern TAG_PATTERN = Pattern.compile("^[a-zA-Z0-9_][a-zA-Z0-9\\-_.]{0,127}$");
+	private static final Pattern DIGEST_PATTERN = Pattern.compile("^[A-Za-z][A-Za-z0-9]*(?:[\\-_+.][A-Za-z][A-Za-z0-9]*)*[:][[\\p{XDigit}]]{32,}$");
```

Comment:
Is this from the standard format (specified somewhere?) for digest similar to tag pattern above?

Reply:
This from the docker's own source code : https://github.com/docker/distribution/blob/master/reference/regexp.go#L44